### ISSUE TITLE
[BUG] Importing breaks on wagmi ^0.8

### ISSUE THREAD

**Issue by danizord** [family/connectkit#52]
**Describe the bug**

When upgrading wagmi from ^0.7 to ^0.8, or installing wagmi without a version constraint as suggested in getting started guide:
![image](https://user-images.githubusercontent.com/1850941/203382859-2bd126b4-f644-4ea5-91ca-9027437f5248.png)

**To reproduce**

https://codesandbox.io/s/connectkit-nextjs-typescript-forked-t3p2j0

**Comment by cte** [comment]
Relevant: https://github.com/family/connectkit/issues/42#issuecomment-1311221683

**Comment by apecollector** [comment]
Can't install it either:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: repro@0.1.0
npm ERR! Found: wagmi@0.8.6
npm ERR! node_modules/wagmi
npm ERR!   wagmi@"^0.8.6" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer wagmi@"^0.6.0" from connectkit@1.0.1
npm ERR! node_modules/connectkit
npm ERR!   connectkit@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /dev/repro/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /dev/repro/.npm/_logs/2022-11-27T04_35_46_895Z-debug-0.log
```

**Comment by lochie** [comment]
ConnectKit is still working from version `0.7.x` of [wagmi](https://wagmi.sh/), a temporary fix is to downgrade wagmi to `0.7.x` for now.
wagmi `0.8.x` has some [breaking changes](https://wagmi.sh/react/migration-guide#08x-breaking-changes) with the module that we have fixed in PR https://github.com/family/connectkit/pull/54, which I'm just waiting for some code reviews before we push up to npm which will hopefully be early this week.

**Comment by lochie** [comment]
We've just released ConnectKit version `1.1.0` which now supports wagmi `0.8.x` 🙌
Thanks for reporting this issue, we appreciate the contribution!
cc @danizord  @apecollector @cte
